docs/index.md

```diff
@@ -18,6 +24,7 @@ Supported features of the specification:
 | AVRO Event Format             | :x: | :x: |
 | HTTP Protocol Binding         | :heavy_check_mark: | :heavy_check_mark: |
 | - [Vert.x](https://github.com/cloudevents/sdk-java/tree/master/http/vertx)        | :heavy_check_mark: | :heavy_check_mark: |
+| - [Jakarta Restful WS](https://github.com/cloudevents/sdk-java/tree/master/http/restful-ws) | :heavy_check_mark: | :heavy_check_mark: |
```

Comment:
`Vert.x` is preceded by a `-`, can you remove it?

Reply:
That's intended :smile: 

![image](https://user-images.githubusercontent.com/6706544/85689310-cef87c00-b6d2-11ea-9da9-03557a295df4.png)

Reply:
Oh, ok, that's fine :smile: